#!/usr/bin/env python
PACKAGE = "ls01d"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("int_param",    int_t,    0, "An Integer parameter", 50,  0, 100)
gen.add("double_param", double_t, 0, "A double parameter",    .5, 0,   1)
gen.add("str_param",    str_t,    0, "A string parameter",  "Hello World")
gen.add("bool_param",   bool_t,   0, "A Boolean parameter",  True)
gen.add("angle_min", double_t, 0, "truncated minimum angle",    -3.1416, -3.1416,   3.1416)
gen.add("angle_max", double_t, 0, "truncated maximum angle",    3.1416, -3.1416,   3.1416)

exit(gen.generate(PACKAGE, "dynamic_param", "param"))
